### PR TITLE
Feature/house delete

### PR DESCRIPTION
**Pull request by nikrb**
this pr adds house delete functionality.

This pr also addresses:
bugs related to crud ops on house/issue
change house/issues to use url navigation.
remove form field default values and replace with placeholders
rework of redux, back to basics - it was getting out of hand and I lost the plot.
fixed lint errors for dashboard, house and issue.

### CODE REVIEW

File: frontend/src/containers/Search/HouseItem.js [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+import {ImageDefault} from '../../components/Image';
 import { css, StyleSheet } from 'aphrodite';
 
 const HouseItem = ({ house }) => (
   <div className={css(styles.container)}>
-    { house.images.length > 0 ?
-      <img className={css(styles.image)} src={house.images[0]} alt="noimg" /> :
-      <img className={css(styles.image)} src="http://via.placeholder.com/350x150" alt="noimg" />
-    }
+    <ImageDefault src={house.images[0]} missing_url="http://via.placeholder.com/350x150" />
```
Comment (nikrb) [inline frontend/src/containers/Search/HouseItem.js]: @vivekimsit this allows uploaded images to be displayed, not just url images

File: frontend/src/containers/Search/actions.js [hunk 2]
```diff
@@ -12,7 +12,9 @@ export function searchHousesSuccess(postCode, json) {
     type: SEARCH_HOUSES_SUCCESS,
     postCode,
     houses: (json.houses || [])
-      .filter(house => house.location.postCode.toLowerCase().includes(postCode.toLowerCase()))
+      .filter(house => house.location.postCode.toLowerCase()
```
Comment (nikrb) [inline frontend/src/containers/Search/actions.js]: @vivekimsit include active flag when querying properties

File: frontend/src/components/Issue/FilteredIssueList.js [hunk 3]
```diff
@@ -13,15 +13,13 @@ export default class FilteredIssueList extends Component {
...
-    console.log( "filtered issue list redirect to issue:", issue);
     this.setState( {redirect: issue});
   };
   render = () => {
     const {data, statusFilter, title} = this.props;
     if( this.state.redirect) {
-      console.log( "filtered issue list issue:", this.state.redirect);
       return <Redirect to={{
-        pathname: "/Issue",
+        pathname: "/issue/"+this.state.redirect._id,
```
Comment (Mortuie) [inline frontend/src/components/Issue/FilteredIssueList.js]: `"/issue/${this.state.redirect._id}"` I think this looks prettier ?

File: frontend/src/components/Uploader/Uploader.js [hunk 4]
```diff
@@ -78,4 +75,34 @@ export default class Uploader extends Component {
...
+    background: "lightgrey",
+    maxWidth: "240px"
+  },
+  well: {
+    margin: "5px auto",
+    // padding: "10px",
+    // paddingBottom: "10px",
+    // marginBottom: "0px"
+  },
+  image_style: { maxWidth: "215px", maxHeight: "215px" },
```
Comment (Mortuie) [inline frontend/src/components/Uploader/Uploader.js]: Don't like this keep it expanded, you don't gain much if anything from this and it breaks consistency.

**Pull request merged**